src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntity.php

```diff
@@ -58,11 +60,19 @@ public function __construct(
         ?array $identifierFieldNames = null,
         ?array $groups = null,
         $payload = null,
-        array $options = [],
+        ?array $options = null,
     ) {
         if (\is_array($fields) && \is_string(key($fields)) && [] === array_diff(array_keys($fields), array_merge(array_keys(get_class_vars(static::class)), ['value']))) {
-            $options = array_merge($fields, $options);
+            trigger_deprecation('symfony/validator', '7.2', 'Passing an array of options to configure the %s constraint is deprecated, use named arguments instead.', static::class);
```

Comment:
```suggestion
            trigger_deprecation('symfony/validator', '7.2', 'Passing an array of options to configure the "%s" constraint is deprecated, use named arguments instead.', static::class);
```

Not sure about this one, but I think this is how messages are formatted when using FQCN?